home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/quartz/jobs/DeactivateApartmentInvitationsJob.java

```diff
@@ -0,0 +1,23 @@
+package com.softserveinc.ita.homeproject.homeservice.quartz.jobs;
+
+import com.softserveinc.ita.homeproject.homeservice.quartz.config.QuartzJobBeanAutoConfiguration;
+import com.softserveinc.ita.homeproject.homeservice.service.impl.ApartmentInvitationServiceImpl;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.quartz.JobExecutionContext;
+import org.springframework.scheduling.quartz.QuartzJobBean;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+@QuartzJobBeanAutoConfiguration(cron = "0 0 11,23 ? * *")
+public class DeactivateApartmentInvitationsJob extends QuartzJobBean {
```

Comment:
As `endTime` property exists in abstract Invitation I think it makes sense to have only one job per this action. Please combine DeactivateApartmentInvitationsJob and DeactivateCooperationInvitationsJob into single job.

Reply:
done